src/i18n/config.json

```
@@ -113,9 +113,18 @@
     "name": "Brazilian Portuguese",
     "langDir": "ltr",
     "dateFormat": "DD.MM.YYYY",
-    "hrefLang": "",
+    "hrefLang": "pt-br",
     "enabled": false
   },
+  {
+    "code": "pt",
```

Think this needs to be expanded to clarify the portugul variant. That's how the Crowdin setup currently has it for the other sites
```suggestion
    "code": "pt-pt",
```

---

@nschonni I am translating the content using a more general
Portuguese that all native Portuguese speakers can understand well.

I did it with the following resources:

- [Nuxt website](https://nuxtjs.org/pt/)
- [Vite website](https://vite-docs-pt.netlify.app)
- [Vue.js 3 website](https://vue3-docs-pt.netlify.app)
- [Pinia website](https://pinia-docs-pt.netlify.app)

And there are more and no native Portuguese person
have complained about them.

---

Just `pt` code should be OK. As it is also a valid ISO-639 code.